public/material-kit/.github/workflows/main.yml

```diff
@@ -0,0 +1,12 @@
+name: Autocloser
```

Comment:
This and many files below should be excluded (all of .github, sources for material-kit). We can check in the compiled css and js only.

Reply:
so should I just completely delete the material-kit/ folder?